orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/AtlasQueueMonitor.kt

```diff
@@ -42,7 +56,11 @@ class AtlasQueueMonitor
 @Autowired constructor(
   private val queue: MonitorableQueue,
   private val registry: Registry,
-  private val clock: Clock
+  private val repository: ExecutionRepository,
+  private val clock: Clock,
+  @Value("\${queue.zombie.check.enabled:false}")private val zombieCheckEnabled: Boolean,
```

Comment:
```
queue:
  zombie:
    check:
      enabled: true
```

Is that what we'd like?

```
queue:
  zombieDetection:
    enabled: true 
    frequency: ...
```

Not sure if ^^ is better or not?

Reply:
ok, mr picky

Reply:
it's not like I suggested putting a space before `private val`